renderer-macos/lib/Brisk.re

```diff
@@ -113,11 +124,12 @@ module UI = {
       let node = root.layoutNode.container;
       let flip = node.context.isYAxisFlipped;
 
+
       Layout.FlexLayout.(
         layoutNode(
           node,
-          Flex.FixedEncoding.cssUndefined,
-          Flex.FixedEncoding.cssUndefined,
+          Flex.FloatEncoding.cssUndefined,
```

Comment:
Let's add a `cssUndefined` to our functor Layout? I think we had it there before.